pycaching/cache.py

```diff
@@ -780,7 +792,17 @@ def load(self):
 
         self.location = Point.from_string(root.find(id="uxLatLon").text)
 
-        self.state = root.find("ul", "OldWarning") is None
+        # Cache status
```

Comment:
I would probably prefer this parsing to happen in a `classmethod` of `Status`, like

```
@classmethod from_cache_details(cls, root):
    ...
```

This would reduce the amount of code for the generic parsing method to make everything more readable.

Reply:
Done in https://github.com/tomasbedrich/pycaching/pull/193/commits/87dcf189a120198f050d8c7f2235e32fcc2ed3e3